data_safe_haven/validators/validators.py

```diff
@@ -124,7 +124,7 @@ def ip_address(ip_address: str) -> str:
     try:
         return str(ipaddress.ip_network(ip_address))
     except Exception as exc:
-        msg = "Expected valid IPv4 address, for example '1.1.1.1'."
+        msg = "Expected valid IPv4 address, for example '1.1.1.1', or 'Internet'."
```

Comment:
```suggestion
        msg = "Expected valid IPv4 address, for example '1.1.1.1', or an Azure ServiceTag, for example 'Internet'."
```

Reply:
As above, only Intenet.

Reply:
OK, but the current wording implies that 'Internet' is a valid IPv4 address. Can you find a better phrasing?

Reply:
Hmm, good point. I _think_ it is correct with the commas, but it is complicated.

Maybe two sentences, "Expected either a valid IPv4 address or 'Internet'. IPv4 addresses should be formatted like '1.1.1.1'.".

Reply:
Actually, I think this is a mistake! This validator _only_ accepts IP addresses.

Reply:
Also fixed in #2247